### ISSUE TITLE
Unable to install dtale through Azure Artifacts feed

### ISSUE THREAD

**Issue by idling-mind** [man-group/dtale#694]
I was trying to install dtale through our internal Azure Artifacts feed and got this error.

```
> python.exe -m pip install dtale
Looking in indexes: https://***:****@***.***.****.internal/DefaultCollection/_packaging/pypackages/pypi/simple/
Collecting dtale
  ERROR: HTTP error 400 while getting https://***.***.***.internal/DefaultCollection/_packaging/9e526b0b-a96f-4467-acd5-9d27dd953a6b/pypi/download/dtale/2.7.1/dtale-2.7.1-py2.py3-none-any.whl#sha256=1d070896e7dee26b388367a6884eb6a619f648c579925f0aa2d4d11de46173d9 (from https://***.***.***.***/DefaultCollection/_packaging/pypackages/pypi/simple/dtale/)
ERROR: Could not install requirement dtale from https://***.***.***.***/DefaultCollection/_packaging/9e526b0b-a96f-4467-acd5-9d27dd953a6b/pypi/download/dtale/2.7.1/dtale-2.7.1-py2.py3-none-any.whl#sha256=1d070896e7dee26b388367a6884eb6a619f648c579925f0aa2d4d11de46173d9 because of HTTP error 400 Client Error: Bad Request - The package description exceeds the maximum length of 324608 bytes. (DevOps Activity ID: 81F972B0-04EE-4CE4-9786-92313E1DF17B) for url: https://***.***.***.***/DefaultCollection/_packaging/9e526b0b-a96f-4467-acd5-9d27dd953a6b/pypi/download/dtale/2.7.1/dtale-2.7.1-py2.py3-none-any.whl for URL https://***.***.***.***/DefaultCollection/_packaging/9e526b0b-a96f-4467-acd5-9d27dd953a6b/pypi/download/dtale/2.7.1/dtale-2.7.1-py2.py3-none-any.whl#sha256=1d070896e7dee26b388367a6884eb6a619f648c579925f0aa2d4d11de46173d9 (from https://***.***.***.***/DefaultCollection/_packaging/pypackages/pypi/simple/dtale/)
```

Based on the sentence `Bad Request - The package description exceeds the maximum length of 324608 bytes`, I am guessint it's because of the long project description.
A similar issue is mentioned here (https://developercommunity.visualstudio.com/t/python-package-cannot-be-installed-from-azure-arti/1434932)

**Comment by aschonfeld** [comment]
So the actual description the package is quite small, but there is a `long_description` field that contains the contents of the README as well as the changelog.  So I can work on trimming that down a bit...

**Comment by aschonfeld** [comment]
@idling-mind just released v2.8.0 with this update.  Please let me know if this works 🙏

**Comment by idling-mind** [comment]
@aschonfeld works now! Closing the issue.
